app/import_test.go

```diff
@@ -278,7 +280,7 @@ func AssertFileIdsInPost(files []*model.FileInfo, th *TestHelper, t *testing.T)
 	}
 }
 
-func TestRewriteFilePaths(t *testing.T) {
+func TestProcessAttachments(t *testing.T) {
```

Comment:
Is there any way to test error paths for `processAttachments`? The new additions don't get hit by the tests.

Reply:
Yeah I can add a couple of test cases :+1: